src/main/resources/LabelsBundle_cy.properties

```diff
@@ -80,7 +80,7 @@ release-date=Dyddiad y datganiad
 next-release=Cyhoeddiad nesaf
 print-full-report=Argraffu adroddiad llawn
 download-pdf=Lawrlwytho pdf
-print-this-statistical-bulletin=Printiwch y bwletin ystadegol hwn
+print-this=Argraffu'r
```

Comment:
According to Google (https://www.google.com/search?q=welsh+translation&rlz=1C5CHFA_enGB865GB865&oq=welsh+tran&aqs=chrome.0.0i433i512j69i57j0i512l8.4788j0j15&sourceid=chrome&ie=UTF-8) the Welsh Argraffu'r translates to "Print the" rather than "Print this" but maybe that's close enough.

Reply:
See https://github.com/ONSdigital/babbage/pull/629 🙂